### PR TITLE
Change to inclusive naming

### PR DESCRIPTION
**Pull request by cristiklein**
See https://inclusivenaming.org/ on why "multi-master" should be avoided.

### CODE REVIEW

**Comment by kbhawkey** [comment]
👋 @cristiklein . Thanks for contributing!

Page preview:
https://deploy-preview-26725--kubernetes-io-master-staging.netlify.app/docs/concepts/overview/components/#control-plane-components

**Comment by kbhawkey** [comment]
@cristiklein . Would you verify that you signed the CLA? Thanks!

**Comment by cristiklein** [comment]
> @cristiklein . Would you verify that you signed the CLA? Thanks!

Done, thanks!

File: content/en/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The control plane's components make global decisions about the cluster (for exam
 Control plane components can be run on any machine in the cluster. However,
 for simplicity, set up scripts typically start all control plane components on
 the same machine, and do not run user containers on this machine. See
-[Building High-Availability Clusters](/docs/admin/high-availability/) for an example multi-master-VM setup.
+[Building High-Availability Clusters](/docs/admin/high-availability/) for an example multi-control-plane-VM setup.
```
Comment (sftim) [inline content/en/docs/concepts/overview/components.md:36]: “multi-control-plane-VM” uses lots of hyphens, and I think could be hard to understand.

We should reword from the current text. How do feel about:
```suggestion
[Creating Highly Available clusters with kubeadm](/docs/setup/production-environment/tools/kubeadm/high-availability/)
for an example control plane setup that runs across multiple VMs.
```

I also fixed the hyperlink.

**Pull request merged**